de.atb.typhondl.xtext.ui/src/de/atb/typhondl/xtext/ui/properties/polystore.properties

```diff
@@ -60,6 +60,7 @@ qlserver.limit.memory =
 qlserver.limit.cpu =
 qlserver.reservation.memory = 
 qlserver.reservation.cpu =
+qlserver.timezone = 
```

Comment:
Why is it s property?  Or are all info always past as a property between the different screens of the wizard?

Reply:
Yes. Also the model creation uses those properties [(here)](https://github.com/typhon-project/typhondl/blob/0640d357641485c10a73b57ce8a22f20cad1c98a/de.atb.typhondl.xtext.ui/src/de/atb/typhondl/xtext/ui/scriptGeneration/DeploymentModelService.java#L166).